### PR TITLE
initial functionality for tap to confirm passphrase words

### PR DESCRIPTION
**Pull request by daviddahl**
This is an initial concept for the "tap to verify passphrase words" #622

No doubt there are some style issues and there are no tests written yet, but I thought I would submit this after a couple hours of hacking.

I tried to change as little as possible, even leaving the full submission form (but hidden). I did not dig in enough yet to see what you are doing to optionally add classes to nodes, etc. I just wanted to get the functionality working.

A question about interaction: Do you want to display any kind of warning when the wrong word is tapped on? For now, it just does nothing, which is clear to me as a nerd that I tapped in the wrong order:)

### CODE REVIEW

**Comment by daviddahl** [comment]
I was looking through the code and do not see any UI tests, perhaps I am not looking in the correct place? I see a bunch of actions/reducer tests only

**Comment by larrysalibra** [comment]
Hey @daviddahl thank you so much for this! I'm just getting over a cold and am a bit behind. I will take a look at this in a couple days.

> I was looking through the code and do not see any UI tests, perhaps I am not looking in the correct place? I see a bunch of actions/reducer tests only

Ha! You're right! We don't currently have UI tests - this is something we need to work on going forward.

**Comment by daviddahl** [comment]
@larrysalibra no worries, get well. I am still thinking about the interaction and what happens after you tap a word. I should have more time tonight

**Comment by daviddahl** [comment]
@larrysalibra I actually had a bit more code to add to this PR. Do you want it to be made against a different branch?

**Comment by larrysalibra** [comment]
I'm sorry about that! We just merged the `feature/redesign` branch into `develop` and github automatically closed all pull requests on that branch. I've re-opened it and set it to be based on `develop`. Feel free to add more code!

File: app/styles/app.css [hunk 1]
```diff
@@ -482,7 +517,7 @@ a.search-result:hover{color:#fff!important;background-color:#2275d7;}
...
-div.ReactModal__Content.ReactModal__Content--after-open{width:415px;background-color:#FAFAFA;background-color:-webkit-linear-gradient(top,#fafafa 0%,#f2f2f2 100%);border:1px solid #FAFAFA;padding:3.75rem 42px;-webkit-box-shadow:0px 0px 25px 0px rgba(0,0,0,0.25);-moz-box-shadow:0px 0px 25px 0px rgba(0,0,0,0.25);box-shadow:0px 0px 25px 0px rgba(0,0,0,0.25);-webkit-border-radius:8px;-moz-border-radius:8px;border-radius:8px;transition:all 300ms ease-in-out;}
+div.ReactModal__Content.ReactModal__Content--after-open{width:470px;background-color:#FAFAFA;background-color:-webkit-linear-gradient(top,#fafafa 0%,#f2f2f2 100%);border:1px solid #FAFAFA;padding:3.75rem 42px;-webkit-box-shadow:0px 0px 25px 0px rgba(0,0,0,0.25);-moz-box-shadow:0px 0px 25px 0px rgba(0,0,0,0.25);box-shadow:0px 0px 25px 0px rgba(0,0,0,0.25);-webkit-border-radius:8px;-moz-border-radius:8px;border-radius:8px;transition:all 300ms ease-in-out;}
```
Comment (daviddahl) [inline app/styles/app.css]: I widened this modal - not sure if that is desirable - I was conflicted between this and using a narrower grid, but that makes the UI longer - again not sure what is best here

File: app/styles/app.css [hunk 2]
```diff
@@ -50,6 +50,41 @@ a.view-out-link{color:#494f56;}
 .fa-green{color:#1d9d74;}
 .app-account-identifier{margin-left:10px;}
 
+.tapped-identity-words {
```
Comment (daviddahl) [inline app/styles/app.css]: I left all of this CSS expanded for review

**Comment by daviddahl** [comment]
@larrysalibra I figured it was something like that. Cool.

**Comment by daviddahl** [comment]
So there are probably still some tweaks to work out in the UI but I feel like - minus tests - this is a good starting point implementation

**Pull request closed without merging**